exercises/gigasecond/README.md

```diff
@@ -4,6 +4,9 @@ Calculate the moment when someone has lived for 10^9 seconds.
 
 A gigasecond is 10^9 (1,000,000,000) seconds.
 
+If you're unsure what operations you can perform on `DateTime<Utc>` take a look at the [chrono crate](https://docs.rs/chrono/0.4.0/chrono/) which is listed as a dependency in the `Cargo.toml` file for this exercise.
```

Comment:
I'm still a bit concerned this isn't a clear enough hint to walk users to adding `Duration` as an import from the `chrono` crate. When I look at the [crate documentation on Duration]:(https://docs.rs/chrono/0.4.0/chrono/#duration)
> Chrono currently uses the time::Duration type from the time crate to represent the magnitude of a time span...
It implies to me as a beginner in Rust that I should be doing the same thing `chrono` does and adding `time` to my `Cargo.toml` so I can also use the `time::Duration` type.

Duration doesn't appear in [reexports](https://docs.rs/chrono/0.4.0/chrono/#reexports) and the function most people end up using for this exercise `Duration::from` isn't apparent anywhere on the page.

Do y'all have any suggestions on how to help this hint provide a bit more structured guidance for people new to the Cargo ecosystem?

Reply:
> I'm still a bit concerned this isn't a clear enough hint to walk users to adding `Duration` as an import from the `chrono` crate.

I see what you mean because there's a big dedicated section about durations being from `time` and being unclear on what relationship this has with https://docs.rs/chrono/0.4.0/chrono/struct.Duration.html.

I don't have any definitive answers, but how about some ideas?

* Shall we link to the relevant `+` operator? https://docs.rs/chrono/0.4.0/chrono/struct.DateTime.html#method.add-1
* Shall we link to the current Duration? https://docs.rs/chrono/0.4.0/chrono/struct.Duration.html